patches/minecraft/net/minecraft/world/World.java.patch

```diff
@@ -560,7 +560,25 @@
              {
                  return false;
              }
-@@ -2011,6 +2167,11 @@
+@@ -1971,7 +2127,16 @@
+                 for (int j4 = j3; j4 < k3; ++j4)
+                 {
+                     IBlockState iblockstate1 = this.func_180495_p(blockpos$pooledmutableblockpos.func_181079_c(l3, i4, j4));
+-
```

Comment:
Please leave this newline in, and change the patch to look like my comment below.